### DISCUSSION TITLE
when re-indexing, do I need to change the vector store DB URI from output\lancedb to update_output\lancedb?

### DISCUSSION THREAD

**Discussion by octadion** [microsoft/graphrag#1366] [Q&A]
Hi everyone,

I'm currently trying out incremental indexing in version 0.4.0. I've been updating data and indexing with update_output set for the index storage. My question is: when re-indexing, do I need to change the vector store DB URI from output\lancedb to update_output\lancedb?

From my tests, if I perform a local query with storage at output, it doesn’t respond with the newly updated data context. However, if I switch the storage to update_output, it responds correctly with the latest data.

Is changing the storage URI necessary for incremental indexing, Or, is it fine to keep the vector DB (lancedb) in output as long as I use the indexed data as the storage (update_output) to ensure the latest data context is available?

thanks

**Comment by JoedNgangmeni** [comment]
Does the incremental update of 0.4.0 allow us to update the knowledge base without having to rebuild the entire index?
